### PR TITLE
test(middleware/metadata): add Test_hasPrefix

### PR DESCRIPTION
**Pull request by 180909**

### CODE REVIEW

File: middleware/metadata/metadata_test.go [hunk 1]
```diff
@@ -137,3 +137,24 @@ func Test_WithPropagatedPrefix(t *testing.T) {
 		t.Error("The prefix must be overridden.")
 	}
 }
+
+func Test_hasPrefix(t *testing.T) {
```
Comment (demoManito) [inline middleware/metadata/metadata_test.go:141]: TestHasPrefix

Comment (demoManito) [reply middleware/metadata/metadata_test.go:141]: Golang does not recommend this naming method, and the standard library does not have this naming method

**Comment by shenqidebaozi** [comment]
fix conflicting files

**Comment by 180909** [comment]
I check the file, and the `TestOptions_hasPrefix` function already exists. So, i close this PR.

**Pull request closed without merging**